### PR TITLE
1.16 compatibility

### LINKED ISSUE #21: [Request] Support for Snapshot versions of WorldEdit for 1.16
Title says it all.
You can close this issue if you're not planning to update yet.

### PR DESCRIPTION
**Pull request by zml2008**
Here's the first pass of updating.

Fabulous graphics still don't work properly, but other modes seem to work fine.

I know there are some issues rendering with optifine -- i wonder if tracking down the layering issues that cause trouble with fabulous will resolve those?

Fixes #21

### CODE REVIEW

**Comment by zml2008** [comment]
Alright, I've sorted out the issues rendering in fabulous graphics -- we now target the transparent blocks frame buffer. At this point from my testing, WorldEditCUI is fully functional on 1.16.

I've done a touch more cleanup to the build file, updated to use newly added Fabric API events, and fixed a config loading issue as well.

File: src/main/java/com/mumfrey/worldeditcui/config/CUIConfiguration.java [hunk 1]
```diff
@@ -29,29 +33,29 @@
...
+	private Colour polyGridColor		  = ConfiguredColour.POLYGRID.getDefault();
+	private Colour polyEdgeColor		  = ConfiguredColour.POLYBOX.getDefault();
+	private Colour polyPointColor		 = ConfiguredColour.POLYPOINT.getDefault();
+	private Colour ellipsoidGridColor	 = ConfiguredColour.ELLIPSOIDGRID.getDefault();
+	private Colour ellipsoidPointColor	= ConfiguredColour.ELLIPSOIDCENTRE.getDefault();
+	private Colour cylinderGridColor	  = ConfiguredColour.CYLINDERGRID.getDefault();
+	private Colour cylinderEdgeColor	  = ConfiguredColour.CYLINDERBOX.getDefault();
+	private Colour cylinderPointColor	 = ConfiguredColour.CYLINDERCENTRE.getDefault();
+	private Colour chunkBoundaryColour	= ConfiguredColour.CHUNKBOUNDARY.getDefault();
+	private Colour chunkGridColour		= ConfiguredColour.CHUNKGRID.getDefault();
```
Comment (mikroskeem) [inline src/main/java/com/mumfrey/worldeditcui/config/CUIConfiguration.java:55]: I think either those equal signs should be aligned properly, or not being aligned at all (just 1 space before/after equals sign - as usually)

File: src/main/java/com/mumfrey/worldeditcui/event/listeners/CUIListenerWorldRender.java [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
+
 	public void onRender(float partialTicks)
 	{
+		Framebuffer buffer = null;
+		if(MinecraftClient.isFabulousGraphicsOrBetter()) {
+			buffer = MinecraftClient.getInstance().worldRenderer.getTranslucentFramebuffer();
+		}
+		if(buffer != null) {
+		    buffer.beginWrite(false);
+        }
```
Comment (mikroskeem) [inline src/main/java/com/mumfrey/worldeditcui/event/listeners/CUIListenerWorldRender.java:38]: Indentation is messed up

File: build.gradle.kts [hunk 3]
```diff
@@ -6,23 +6,38 @@ plugins {
 group = "com.mumfrey.worldeditcui"
 version = "1.16.1_01"
 
+repositories {
+    maven(url="https://maven.sk89q.com/repo") {
```
Comment (mikroskeem) [inline build.gradle.kts:10]: Whitespace, `maven(url = "https://...") {`

**Comment by zml2008** [comment]
formatting changes fixed -- just tossed in another change to versioning so fabric loader stops giving warnings -- going for minimal change, the mod version is its MC version, and the "build" field is the revision of the mod.

**Comment by mikroskeem** [comment]
LGTM, will merge later today.

**Comment by mikroskeem** [comment]
Thank you

**Pull request merged**